src/api-reference/direct-connects/hotel-service-2/Availability.markdown

```diff
@@ -300,6 +318,7 @@ For a description of the relationship between the `RoomID` and `RatePlanID` refe
 |`CancelPenalties`|`complex`|**Required if `RateDetailsInd` is `true` or not present** Collection of cancellation penalties. If the cancel penalties are not provided SAP Concur will display: "Cancellation policy not provided by vendor".|
 |`MealsIncluded`|`complex`|Defines which meals are included with this rate program.|
 |`RatePlanDescription`|`complex`|Textual information regarding the Rate Plan.|
+|`RatePlanType`|`integer`|**Optional** specify the rate plan type of this rate. Refer to Open Travel Code List Rate Plan Type (RPT)|
```

Comment:
CWT is overloading this to indicate refunable rates, will need to get them to change for T2

Reply:
It's not common construct in our docs to identify `**Optional**`

Reply:
@rlee-sap  - i do think it makes the doc more clear, but agree it's not common in other parts. will remove

Reply:
done, please review again

Reply:
@rlee-sap